src/components/CarouselItem.js

```diff
@@ -1,15 +1,21 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import classname from 'classnames';
 import '../styles/CarouselItem.scss';
 
 export default class CarouselItem extends Component {
   static propTypes = {
+    onClick: PropTypes.func,
     children: PropTypes.node,
     width: PropTypes.number,
   };
   render() {
     return (
-      <li className="BrainhubCarouselItem" style={{ width: `${this.props.width}px` }}>
+      <li
+        className={classname('BrainhubCarouselItem', { 'BrainhubCarouselItem--clickable': this.props.onClick })}
+        style={{ width: `${this.props.width}px` }}
+        onClick={this.props.onClick}
```

Comment:
You can create method for onClick and call it there with passed index as separate props.
```
clickHandler = () => this.props.onClick(this.props.index);
```
And in parent component:
```
 itemOnClickHandler = index => this.changeSlide(index);
```
So that you can avoid creating new function every render

Reply:
Will be done in future PR